apps/services/endorsements/api/scripts/updateMetadata.ts

```diff
@@ -1,168 +1,272 @@
-import 'reflect-metadata'
-import { NestFactory } from '@nestjs/core/nest-factory'
-import { Sequelize } from 'sequelize-typescript'
-import Bottleneck from 'bottleneck'
-import * as sequelizeConfig from '../sequelize.config'
-import { Endorsement } from '../src/app/modules/endorsement/models/endorsement.model'
-import { EndorsementList } from '../src/app/modules/endorsementList/endorsementList.model'
-import { EndorsementMetadataService } from '../src/app/modules/endorsementMetadata/endorsementMetadata.service'
-import { EndorsementMetaField } from '../src/app/modules/endorsementMetadata/types'
-import { AppModule } from '../src/app/app.module'
-import { EndorsementMetadata } from '../src/app/modules/endorsementMetadata/endorsementMetadata.model'
-import { logger } from '@island.is/logging'
-
-interface EndorsementListFieldMap {
-  [key: string]: EndorsementMetaField[]
-}
+// import 'reflect-metadata'
```

Comment:
so can we remove this whole file? seems like we can just bring it back form git history if we need it

Reply:
Removed the file and all mentions of it in other files.